readme/template-readme.md

```diff
@@ -0,0 +1,101 @@
+# README Template
+
+*{Before using this template, read the accompanying README template guide}.*
+
+## {Project} logo and badges
+{This section is optional. Share your logo and add relevant badges to your README.}
+
+## {Project Name} 
+{Optional: Include the project URL and the project owner name}
+
+## Table of contents 
+{This section is optional.}
+1. Heading 1
+2. Heading 2
+3. Heading 3...
+
+## Project description 
+{Project description example available in README template guide}
+
+With *{Project Name}* you can *(verb)* *(noun)*...
+
+*{Project Name}* helps you *(verb)* *(noun)*....
+
+Unlike *(alternative)*, *{Project Name}* *(verb)* *(noun)*... 
+
+{Optional: Include screenshots and/or demo videos}
+
+## Who this project is for 
+This project is intended for {target audience} who want to {audience objective}. 
+
+## Project dependencies
+To use {Project Name}, you will need: 
+- Prerequisite 1 
+- Prerequisite 2
+- Prerequisite 3...
+
+## Instructions for using {Project Name}
+Get started with {Project Name} by {first step here}
+
+### How to get {Project Name}
```

Comment:
**Question (non-blocking):** For me, I wonder if "install" would be better? We could make it more flexible with a format like this too:

How to get/install/download {Project Name} to show that there's a few choices?

Edited to add: Actually, I see you have an install, configure, run section later. What is the difference between those sections and the "how to get" section in your mind?

Reply:
The install/configure/run was what was in Sachin's original template. I think they likely could be joined together, but I can see the value in breaking it down if a project has a large number of steps for each. Let's chat about this!